jekyll/_cci2/release/manage-releases.adoc

```diff
@@ -0,0 +1,158 @@
+---
+contentTags:
+  platform:
+  - Cloud
+---
+= Manage releases - Open preview
+:page-layout: classic-docs
+:page-liquid:
+:page-description: How to guide for managing releases from the CircleCI web app
+:icons: font
+:experimental:
+
+NOTE: CircleCI releases is currently in **open preview**. During open preview release features are available for free. Charges will be associated with release features once they are generally available.
+
+The steps outlined on this page guide you through the tasks required to manage your releases.
+
+[#prerequisites]
+== Prerequisites
+
+Before following the steps below, ensure the release environment has been set up and configured successfully. Refer to the xref:set-up-a-release-environment#[Set up a release environment] and xref:configure-your-kubernetes-component#[Configure your Kubernetes component] pages.
+
+[#restore-a-version]
+== Restore a version
+
+The _restore a version_ command restores a previously released version of a component.
+
+image::../../img/docs/releases/restore-a-version.png[Screenshot showing the restore a version button]
+
+[#restore-steps]
+=== Steps
+
+. Select btn:[Releases] in the CircleCI web app sidebar
+. You are now in the timeline view. Select your component from the dropdown menu.
+. Scroll to locate the version your want to restore, and click the restore icon in the **Actions** column
+. Read through the recommendations in the popup, and if you want to proceed type `RESTORE`, then click btn:[Restore]
```

Comment:
~ This can also be done from a release details page; there's talk of possibly removing the "restore" button from the timeline view but I don't think we've created a ticket for that yet.

Reply:
Right I think I'll update this then, and make leave a note that this can be done from the timeline view also, which can be removed later